src/DependencyInjection/KnpMenuExtension.php

```diff
@@ -52,15 +52,15 @@ public function load(array $configs, ContainerBuilder $container)
     /**
      * {@inheritdoc}
      */
-    public function getNamespace()
+    public function getNamespace(): string
```

Comment:
this should be done as a separate PR IMO, as it is a totally separate change (a valid one, but unrelated to deprecating these options)

Reply:
I did it because that missing return type is generating a deprecation on latest versions of deps

Reply:
So, it was just an attempt to avoid failing test. But I see that test is still failing, for unrelated changes (that are object of another PR), so I guess we can accept such failure